SoundSwitch/Framework/Configuration/SoundSwitchConfiguration.cs

```diff
@@ -167,6 +167,11 @@ public bool Migrate()
                 NotificationSettings = NotificationTypeEnum.BannerNotification;
             }
 
+            if (NotificationSettings == NotificationTypeEnum.CustomNotification)
+            {
+                NotificationSettings = NotificationTypeEnum.SoundNotification;
```

Comment:
Need to tell that there has been a change in the configuration and that it need to be saved again.
```suggestion
                NotificationSettings = NotificationTypeEnum.SoundNotification;
                migrated = true;
```

The previous line you copied and pasted didn't do it, and that's on me ^^'